scripts/qa_crowbarsetup.sh

```diff
@@ -1715,6 +1723,10 @@ function custom_configuration()
             fi
         ;;
         database)
+            if [ -n "$want_sles12_controller" ] && iscloudver 6plus ; then
```

Comment:
Looks good to me; now we can drop all the ` && iscloudver 6plus`, which makes the code cleaner :-)